
try:
    resultado = 10 / 0
except:
    print("Ocorreu um erro!")

